acceptance/framework/config/config.go

```diff
@@ -22,16 +22,67 @@ const (
 	LicenseSecretKey  = "key"
 )
 
-// TestConfig holds configuration for the test suite.
-type TestConfig struct {
-	Kubeconfig    string
+type KubeEnv struct {
```

Comment:
I think having this as `TestConfig` instead of `KubeEnv` is better as it clashes with KubernetesEnvironment from environment.go.  Either that or merge the two types together?

Reply:
`KubeTestConfig` makes the most sense I guess?

Reply:
I'd be fine with `KubeTestConfig` although the `kube` part is arguably implied by the repo name. 

For context, originally I was thinking that this framework could be reused for other non-kube setups but that never happened. So it probably could make sense to make names more specific.

Reply:
Ah thanks for the history on that, yeah I think specific works in this case since the use case has narrowed